tools/packaging/kata-deploy/Dockerfile

```diff
@@ -18,8 +18,7 @@ RUN \
 apt-get update && \
 apt-get install -y --no-install-recommends apt-transport-https ca-certificates curl xz-utils systemd && \
```

Comment:
Do we install `gpg` here, too?

Reply:
Do we need only `gpg` or the full `gnupg` combo?